README.md

```diff
@@ -54,42 +43,71 @@ Following are a list of modifiable paths:
+  * Input requires a parameter containing the following:
+    * `patient` - Patient FHIR Resource, must include `medication` with `patient`
+    * `medication` - Medication or MedicationRequest FHIR Resource, must include `patient` with `medication`
+    * `authNumber` - String containing the REMS Authorization Number, may be sent without `patient` or `medication`
+  * Returns a GuidanceResponse within a Parameter with the status
+    * Contains Nested GuidanceResponse resources for each ETASU requirement with their status
+* `/4_0_0/Questionnaire/\<form-name\>/$questionnaire-package` - The endpoint for the FHIR Operation used for retrieving the Questionnaire package for a given form
+  * Example: /4_0_0/Questionnaire/TIRFRemsPatientEnrollment/$questionnaire-package
+  * This includes the Questionnaire and any other necessary FHIR resources needed for loading the quesetionnaire form with the REMS SMART on FHIR app
 
-To reach the CDS Services discovery information:
+## Environment Variables
 
-```
-http://localhost:8090/cds-services
-```
+### How To Override Defaults
+
+The .env file contains the default URI paths, which can be overwritten from the start command as follows:
+a) `REACT_APP_LAUNCH_URL=http://example.com PORT=6000 npm start` or b) by specifying the environment variables and desired values in a `.env.local`.
+
+> **Bug**: Do note that the `SMART_ENDPOINT` environment variable cannot be overwritten in a `.env.local`, it must be done in the `.env`.
+
+Following are a list of modifiable paths:
+
+| URI Name        | Default                                    | Description                                                                                           |
+| --------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------|
+| AUTH_SERVER_URI | `http://localhost:8090`                    | The base url of the auth server, currently set to the base url of this app.                           |
+| HTTPS_CERT_PATH | `server.cert`                              | Path to a certificate for encryption, allowing HTTPS. Unnecessary if using HTTP.                      |
+| HTTPS_KEY_PATH  | `server.key`                               | Path to a key for encryption, allowing HTTPS. Unnecessary if using HTTP.                              |
+| LOGGING_LEVEL   | `debug`                                    | Amount to output in the log, can be changed to verbose, info, warn, or error.                         |
+| MONGO_DB_NAME   | `remsadmin`                                | Name of the database table being used. Should be changed if not using the Mongo instructions below.   |                 
+| MONGO_URL       | `mongodb://rems-user:pass@127.0.0.1:27017` | URL for the connection to the database, should be changed if not using the Mongo instructions below.  |
```

Comment:
The mongo instructions are above, in this case

Reply:
updated